package/com.unity.formats.usd/Tests/Performance/PointInstancerPerformanceTests.cs

```diff
@@ -0,0 +1,113 @@
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+using System.IO;
+
+namespace Unity.Formats.USD.Tests
+{
+    [Category("Performance")]
+    public class PointInstancerPerformanceTests : PerformanceBaseFixture
+    {
+        public enum InstancerSize
+        {
+            Small,
+            Med,
+            Large,
+            Larger
+        }
+
+        const string k_PointInstancerPrim10GUID = "ddba66229ec63364d8ce36112afcdbec";
+        const string k_PointInstancerPrim100GUID = "567d2c32dba4a364bb1fb80b134dc92d";
+        const string k_PointInstancerPrim1000GUID = "9ca154d52e7e03e4c989b66e33e08212";
+        // For Import as Prefab / Timeline it takes more than 3 minutes to run the test, so it wont be included there
+        const string k_PointInstancerPrim10000GUID = "b4f2b0c08e1cc0d43aa738154881fcfc";
+
+        private string DeterminePointInstancerSize(InstancerSize testSize)
```

Comment:
Something like `GetGUIDForPointInstancerSize` might be a more accurate name for what this function is doing?

Reply:
Changed to GetGUIDForTestPointInstancerSize!